packages/app/cypress/e2e/specs.cy.ts

```diff
@@ -619,6 +619,80 @@ describe('App: Specs', () => {
       })
     })
 
+    context('Create from component card', () => {
+      beforeEach(() => {
+        cy.scaffoldProject('no-specs-vue-2')
+        cy.openProject('no-specs-vue-2')
+        cy.startAppServer('component')
+        cy.visitApp()
+
+        cy.findAllByTestId('card').eq(0).as('ComponentCard')
+      })
+
+      it('Shows create from component card for Vue projects with default spec patterns', () => {
+        cy.get('@ComponentCard')
+        .within(() => {
+          cy.findByRole('button', {
+            name: 'Create from component',
+          }).should('be.visible')
+          .and('not.be.disabled')
+        })
+      })
+
+      it('Can be closed with the x button', () => {
+        cy.get('@ComponentCard').click()
+
+        cy.findByRole('button', { name: 'Close' }).as('DialogCloseButton')
+
+        cy.get('@DialogCloseButton').click()
+        cy.findByRole('dialog', {
+          name: 'Choose a component',
+        }).should('not.exist')
+      })
+
+      it('Lists Vue components in the project', () => {
+        cy.get('@ComponentCard').click()
+
+        cy.findByText('2 Matches').should('be.visible')
+
+        cy.findByText('App').should('be.visible')
+        cy.findByText('HelloWorld').should('be.visible')
+      })
+
+      it('Allows for the user to search through their components', () => {
+        cy.get('@ComponentCard').click()
+
+        cy.findByText('*.vue').should('be.visible')
+        cy.findByText('2 Matches').should('be.visible')
+        cy.findByLabelText('file-name-input').type('HelloWorld')
+
+        cy.findByText('HelloWorld').should('be.visible')
+        cy.findByText('1 of 2 Matches').should('be.visible')
+        cy.findByText('App').should('not.exist')
+      })
+
+      it('shows success modal when component spec is created', () => {
```

Comment:
Would be cool to test that the spec gets compiled and run after the spec creation.

Reply:
It doesn't look like the test actually runs in Cypress-in-Cypress... this is what I see when I try to run the generated test. Works fine outside of the c-in-c environment

![Screen Shot 2022-08-08 at 2 29 40 PM](https://user-images.githubusercontent.com/14275198/183488276-55dfbdfa-8994-497f-9ec6-f4c38b1a8df4.png)

here's the whole error

```
Error: Child compilation failed:
  Module not found: Error: Can't resolve '/private/var/folders/z_/9smbmcx124zc8d  677fn1pjm40000gq/T/cy-projects/no-specs-vue-2/cypress/support/component-index.  html' in '/private/var/folders/z_/9smbmcx124zc8d677fn1pjm40000gq/T/cy-projects  /no-specs-vue-2'
  ModuleNotFoundError: Module not found: Error: Can't resolve '/private/var/fold  ers/z_/9smbmcx124zc8d677fn1pjm40000gq/T/cy-projects/no-specs-vue-2/cypress/sup  port/component-index.html' in '/private/var/folders/z_/9smbmcx124zc8d677fn1pjm  40000gq/T/cy-projects/no-specs-vue-2'
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/w  ebpack/lib/Compilation.js:2016:28
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/w  ebpack/lib/NormalModuleFactory.js:798:13
      at eval (eval at create (/private/tmp/cy-system-tests-node-modules/no-spec  s-vue-2/node_modules/tapable/lib/HookCodeFactory.js:33:10), :10:1)
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/w  ebpack/lib/NormalModuleFactory.js:270:22
      at eval (eval at create (/private/tmp/cy-system-tests-node-modules/no-spec  s-vue-2/node_modules/tapable/lib/HookCodeFactory.js:33:10), :9:1)
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/w  ebpack/lib/NormalModuleFactory.js:434:22
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/w  ebpack/lib/NormalModuleFactory.js:120:11
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/w  ebpack/lib/NormalModuleFactory.js:670:25
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/w  ebpack/lib/NormalModuleFactory.js:855:8
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/w  ebpack/lib/NormalModuleFactory.js:975:5
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/n  eo-async/async.js:6883:13
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/w  ebpack/lib/NormalModuleFactory.js:958:45
      at finishWithoutResolve (/private/tmp/cy-system-tests-node-modules/no-spec  s-vue-2/node_modules/enhanced-resolve/lib/Resolver.js:312:11)
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/e  nhanced-resolve/lib/Resolver.js:386:15
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/e  nhanced-resolve/lib/Resolver.js:435:5
      at eval (eval at create (/private/tmp/cy-system-tests-node-modules/no-spec  s-vue-2/node_modules/tapable/lib/HookCodeFactory.js:33:10), :16:1)
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/e  nhanced-resolve/lib/Resolver.js:435:5
      at eval (eval at create (/private/tmp/cy-system-tests-node-modules/no-spec  s-vue-2/node_modules/tapable/lib/HookCodeFactory.js:33:10), :27:1)
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/e  nhanced-resolve/lib/DescriptionFilePlugin.js:87:43
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/e  nhanced-resolve/lib/Resolver.js:435:5
      at eval (eval at create (/private/tmp/cy-system-tests-node-modules/no-spec  s-vue-2/node_modules/tapable/lib/HookCodeFactory.js:33:10), :15:1)
      at /private/tmp/cy-system-tests-node-modules/no-specs-vue-2/node_modules/e  nhanced-resolve/lib/Resolver.js:435:5
  
  - Compilation.js:2016 
    [no-specs-vue-2]/[webpack]/lib/Compilation.js:2016:28
  
  - NormalModuleFactory.js:798 
    [no-specs-vue-2]/[webpack]/lib/NormalModuleFactory.js:798:13
  
  
  - NormalModuleFactory.js:270 
    [no-specs-vue-2]/[webpack]/lib/NormalModuleFactory.js:270:22
  
  
  - NormalModuleFactory.js:434 
    [no-specs-vue-2]/[webpack]/lib/NormalModuleFactory.js:434:22
  
  - NormalModuleFactory.js:120 
    [no-specs-vue-2]/[webpack]/lib/NormalModuleFactory.js:120:11
  
  - NormalModuleFactory.js:670 
    [no-specs-vue-2]/[webpack]/lib/NormalModuleFactory.js:670:25
  
  - NormalModuleFactory.js:855 
    [no-specs-vue-2]/[webpack]/lib/NormalModuleFactory.js:855:8
  
  - NormalModuleFactory.js:975 
    [no-specs-vue-2]/[webpack]/lib/NormalModuleFactory.js:975:5
  
  - async.js:6883 
    [no-specs-vue-2]/[neo-async]/async.js:6883:13
  
  - NormalModuleFactory.js:958 
    [no-specs-vue-2]/[webpack]/lib/NormalModuleFactory.js:958:45
  
  - Resolver.js:312 finishWithoutResolve
    [no-specs-vue-2]/[enhanced-resolve]/lib/Resolver.js:312:11
  
  - Resolver.js:386 
    [no-specs-vue-2]/[enhanced-resolve]/lib/Resolver.js:386:15
  
  - Resolver.js:435 
    [no-specs-vue-2]/[enhanced-resolve]/lib/Resolver.js:435:5
  
  
  - Resolver.js:435 
    [no-specs-vue-2]/[enhanced-resolve]/lib/Resolver.js:435:5
  
  
  - DescriptionFilePlugin.js:87 
    [no-specs-vue-2]/[enhanced-resolve]/lib/DescriptionFilePlugin.js:87:43
  
  - Resolver.js:435 
    [no-specs-vue-2]/[enhanced-resolve]/lib/Resolver.js:435:5
  
  
  - Resolver.js:435 
    [no-specs-vue-2]/[enhanced-resolve]/lib/Resolver.js:435:5
  
  - child-compiler.js:169 
    [no-specs-vue-2]/[html-webpack-plugin]/lib/child-compiler.js:169:18
  
  - Compiler.js:551 finalCallback
    [no-specs-vue-2]/[webpack]/lib/Compiler.js:551:5
  
  - Compiler.js:577 
    [no-specs-vue-2]/[webpack]/lib/Compiler.js:577:11
  
  - Compiler.js:1196 
    [no-specs-vue-2]/[webpack]/lib/Compiler.js:1196:17
  
  
  - Hook.js:18 Hook.CALL_ASYNC_DELEGATE [as _callAsync]
    [no-specs-vue-2]/[tapable]/lib/Hook.js:18:14
  
  - Compiler.js:1192 
    [no-specs-vue-2]/[webpack]/lib/Compiler.js:1192:33
  
  - Compilation.js:2787 finalCallback
    [no-specs-vue-2]/[webpack]/lib/Compilation.js:2787:11
  
  - Compilation.js:3092 
    [no-specs-vue-2]/[webpack]/lib/Compilation.js:3092:11
  
  
  - Hook.js:18 Hook.CALL_ASYNC_DELEGATE [as _callAsync]
    [no-specs-vue-2]/[tapable]/lib/Hook.js:18:14
  
  - Compilation.js:3085 
    [no-specs-vue-2]/[webpack]/lib/Compilation.js:3085:38
  
  
  - Compilation.js:492 fn
    [no-specs-vue-2]/[webpack]/lib/Compilation.js:492:9
  
  
  
  - task_queues:96 processTicksAndRejections
    node:internal/process/task_queues:96:5
```